### PR TITLE
Remove deprecated `PlatformMenuBar.body`

### PR DESCRIPTION
**Pull request by gspencergoog**
Part of https://github.com/flutter/flutter/issues/139243

## Description

This removes the `PlatformMenuBar.body` attribute and constructor parameter, since its deprecation period has elapsed.

## Tests
 - No tests were using the deprecated attribute, so no tests were removed.

#FlutterDeprecations

### CODE REVIEW

**Comment by stuartmorgan** [comment]
test-exempt: removing code

File: packages/flutter/test_fixes/widgets/widgets.dart [hunk 1]
```diff
@@ -170,4 +170,7 @@ void main() {
     clipBehavior: Clip.none,
   );
   final Clip clip = details.clipBehavior;
+
```
Comment (Piinks) [inline packages/flutter/test_fixes/widgets/widgets.dart:173]: We'll need to add the same fix tests for material and cupertino imports, there are subdirectories for each under test_fixes.

In the actual fix definition, the `uris` that it applies to map to the import statements which indicate the fix could be relevant for this file.

**Review by Piinks — COMMENTED** [review]
Woohoo! Thanks for adding a dart fix! We have not announced the next round of deprecations, so let's wait to land until next week. 👍

File: packages/flutter/test_fixes/widgets/widgets.dart (hunk 1 above)
Comment (gspencergoog) [reply packages/flutter/test_fixes/widgets/widgets.dart:173]: Done.

**Review by Piinks — APPROVED** [review]
Flutter_LGTM

**Pull request merged**